e2e_test/sink/aggregation.slt

```diff
@@ -0,0 +1,17 @@
+statement ok
+CREATE MATERIALIZED SOURCE s (v1 int) WITH ('connector' = 'datagen') ROW FORMAT json;
+
+statement ok
+CREATE MATERIALIZED VIEW mv AS SELECT avg(v1) AS average FROM s;
+
+statement ok
+CREATE SINK si FROM mv WITH ('kafka.brokers' = '127.0.0.1:29092', 'kafka.topic' = 'test_topic', 'sink.type' = 'append_only', 'sink_type' = 'kafka');
```

Comment:
What does `append_only` mean here? The output from HashAgg doesn't seem to be append-only...